avalanche/training/regularization.py

```diff
@@ -126,4 +128,66 @@ def update(self, experience, model):
                 ].union(pc)
 
 
-__all__ = ["RegularizationMethod", "LearningWithoutForgetting"]
+class ACECriterion(RegularizationMethod):
+    """
+    Asymetric cross-entropy (ACE) Criterion used in
+    "New Insights on Reducing Abrupt Representation 
+    Change in Online Continual Learning"
+    by Lucas Caccia et. al.
+    https://openreview.net/forum?id=N8MaByOzUfb
+    """
+
+    def __init__(
+        self,
+        initial_old_classes: List[int] = None,
+        initial_new_classes: List[int] = None,
+    ):
+        """
+        param: initial_old_classes: List[int]
+        param: initial_new_classes: List[int]
+        """
+        self.old_classes = (
+            set(initial_old_classes) if 
+            initial_old_classes is not None else set()
+        )
+        self.new_classes = (
+            set(initial_new_classes) if 
+            initial_new_classes is not None else set()
+        )
+
+    def update(self, batch_y):
```

Comment:
update will be called `after_train_exp`, so you can use the `experience` or `dataset` to extract the active units.